build.py

```diff
@@ -67,6 +67,7 @@ def set_properties_unit(project):
     project.set_property("dir_source_unittest_python", "tests/lava")
     project.set_property("dir_source_main_scripts", "scripts")
     project.set_property("dir_docs", "docs")
+    project.build_depends_on("lava", url="git+https://github.com/lava-nc/lava.git")
```

Comment:
Before we release lava-dl we should update this to the released lava version that supports netx.